android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/Cloudlet.java

```diff
@@ -551,6 +563,30 @@ protected void onPostExecute(Void result) {
         }
     }
 
+    private class LatencyTestTaskNetTest extends AsyncTask<Void, Void, Void> {
+        @Override
+        protected Void doInBackground(Void... params) {
+            NetTest netTest = new NetTest();
+            Site site = new Site(mContext, NetTest.TestType.CONNECT, mNumPackets, mHostName, mOpenPort);
+            netTest.addSite(site);
+            netTest.testSites(netTest.TestTimeoutMS);
+            latencyMin = 0;
+            latencyAvg = site.average;
+            latencyMax = 0;
+            latencyStddev = site.stddev;

```

Comment:
If you are posting these latency results to EdgeEvents, you could set these stats after the EVENT_LATENCY_PROCESSED event. This will provide min, max, avg, and std dev. Otherwise, you probably have to iterate through the sites.samples to find the min and max (the array should only have length 3 or 5, I think)

Reply:
No, in this case EVENT_LATENCY_PROCESSED is not involved. This is a manual test initiated by the user on the "Cloudlet Details" page. However, this does bring up a good point: Even though this is user-initiated, should we still post the results via `postLatencyResult()`?